src/main/java/radiounit/AbstractManagedRadioUnitRegistry.java

```diff
@@ -0,0 +1,34 @@
+package radiounit;
+
+import java.util.List;
+import common.AlarmStatusLevel;
+import common.Carrier;
+import common.FrequencyBand;
+import common.RatType;
+import common.Vendor;
+
+/**
+ * Abstract class for ManagedRadioUnitRegistry
+ * 
+ * @author esiumat
+ *
+ */
+public abstract class AbstractManagedRadioUnitRegistry {
+
+	List<ManagedRadioUnit> radioUnits;
+
+	public abstract void addRadioUnit(String ipAddress, String name, Vendor vendor);
+
```

Comment:
I like all these methods below! We could look into providing a more sophisticated method, where you pass some combination of vendor, RAT type, alarm status and state, and get back a list of radios that meet all the criteria.

Maybe for now, we could add a method with the following signature that serves that purpose, and potentially revise after:

`public abstract List<ManagedRadioUnit> getRadio(Venor v, RatTypee r, AlarmStatusLevel a, RadioUnitState s);`

Reply:
Hmm, okay. I think that makes sense. Would the use case be for when you don't recall what the name is?

Reply:
I clarified this with David